packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -33,225 +27,52 @@ export default class CSSEditor extends Foundation<
         managedClasses: void 0,
     };
 
-    private spacingPropertyNames: string[] = [
-        SpacingProperty.marginBottom,
-        SpacingProperty.marginLeft,
-        SpacingProperty.marginRight,
-        SpacingProperty.marginTop,
-        SpacingProperty.paddingBottom,
-        SpacingProperty.paddingLeft,
-        SpacingProperty.paddingRight,
-        SpacingProperty.paddingTop,
-    ];
-
-    private postionPropertyNames: string[] = [
-        "position",
-        Location.left,
-        Location.right,
-        Location.top,
-        Location.bottom,
-    ];
-
-    private heightPropertyNames: string[] = ["height"];
-
-    private widthPropertyNames: string[] = ["width"];
-
-    private backgroundPropertyNames: string[] = ["background"];
+    /**
+     * The custom control plugins used in the form
+     */
+    private controlPlugins: StandardControlPlugin[];
 
-    private colorPropertyNames: string[] = ["color"];
+    constructor(props: CSSEditorProps) {
+        super(props);
 
-    private borderPropertyNames: string[] = ["borderColor", "borderStyle", "borderWidth"];
+        const exampleData: any = getDataFromSchema(cssEditorSchema);
 
-    private boxShadowPropertyNames: string[] = ["boxShadow"];
+        this.state = {
+            data: exampleData,
+        };
 
-    private borderRadiusPropertyNames: string[] = [
-        "borderRadius",
-        "borderTopLeftRadius",
-        "borderTopRightRadius",
-        "borderBottomRightRadius",
-        "borderBottomLeftRadius",
-    ];
+        this.controlPlugins = [
+            new StandardControlPlugin({
+                id: "color",
```

Comment:
These strings should be imported constants and used in both this file and the schema to ensure they are in sync.